src/app/Fake.DotNet.NuGet/NuGet.fs

```diff
@@ -392,7 +404,12 @@ let NuGetPack setParams nuspecOrProjectFile =
 let NuGetPublish setParams =
     let parameters = NuGetDefaults() |> setParams
     use __ = Trace.traceTask "NuGet-Push" (packageFileName parameters)
-    publish parameters
+    try
+        publish parameters
+    with exn ->
+        if exn.InnerException <> null then exn.Message + "\r\n" + exn.InnerException.Message else exn.Message
+        |> replaceAccessKeys parameters
+        |> failwith
```

Comment:
Yep, probably we should find more graceful solution as I also don't like those `e. Message + e.InnerException.Message`. However, currently I decided to make this PR simple and just copy the missing changes.